Mixer.py

```diff
@@ -25,6 +28,15 @@ def mix(self, number_of_records=1000, with_incremental_id=False):
             result.append(line)
         return result
 
+    def json(self, number_of_records=30):
```

Comment:
json() method is almost identical to mix(). does it makes sense to reuse mix() in one way or another? 
For example:
* Mixer constructor can accept parameter `m=Mixer(format="json"), m=Mixer(format="csv")`
* mix() method can accept format parameter: `m.mix(number_of_records=10, format="json")`